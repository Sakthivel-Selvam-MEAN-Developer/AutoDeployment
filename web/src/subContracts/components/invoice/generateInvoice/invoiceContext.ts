import { createContext } from 'react'
import { invoiceValuesProps } from './list'
export interface contextProps {
    filterData: filterDataProps
    setFilterData: React.Dispatch<React.SetStateAction<filterDataProps>>
}
export interface filterDataProps {
    pageName: string
    startDate: number
    endDate: number
    cementCompany: { id: number | undefined; name: string | undefined }
}
export const invoiceFilterData = createContext<contextProps>({} as contextProps)
interface billNoContextProps {
    setInvoiceValues: React.Dispatch<React.SetStateAction<invoiceValuesProps>>
    invoiceValues: invoiceValuesProps
}
export const billNoContext = createContext<billNoContextProps>({} as billNoContextProps)
export interface partyNamesProps {
    invoiceNumber: string
    partyName: string
}
