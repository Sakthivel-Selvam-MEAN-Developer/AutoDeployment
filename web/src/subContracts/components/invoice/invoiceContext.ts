import { createContext } from 'react'
import { Nullable } from '../../../types'
import { invoiceValuesProps } from './list'
export interface contextProps {
    filterData?: Nullable<filterDataProps>
    setFilterData?: React.Dispatch<React.SetStateAction<Nullable<filterDataProps>>>
}
export interface filterDataProps {
    pageName: string
    startDate: number
    endDate: number
    cementCompanyName: string
}
export const invoiceFilterData = createContext<Nullable<contextProps | any>>(null)
interface billNoContextProps {
    setInvoiceValues: React.Dispatch<React.SetStateAction<invoiceValuesProps>>
    invoiceValues: invoiceValuesProps
}
export const billNoContext = createContext<billNoContextProps>({} as billNoContextProps)
