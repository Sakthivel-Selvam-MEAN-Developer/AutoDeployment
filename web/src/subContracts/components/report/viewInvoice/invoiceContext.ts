import { createContext } from 'react'
export interface contextProps {
    filterData: filterDataProps
    setFilterData: React.Dispatch<React.SetStateAction<filterDataProps>>
}
export interface filterDataProps {
    pageName: string
    startDate: number
    endDate: number
    cementCompany: { id: number | undefined; name: string | undefined }
    pageNumber: number
    received: boolean | undefined
    GSTReceived: boolean | undefined
}
export const invoiceFilterData = createContext<contextProps>({} as contextProps)
interface billNoContextProps {
    setInvoiceValues: React.Dispatch<React.SetStateAction<invoiceValuesProps>>
    invoiceValues: invoiceValuesProps
}
export const billNoContext = createContext<billNoContextProps>({} as billNoContextProps)
export interface invoiceValuesProps {
    billNo: string
    date: number
}
