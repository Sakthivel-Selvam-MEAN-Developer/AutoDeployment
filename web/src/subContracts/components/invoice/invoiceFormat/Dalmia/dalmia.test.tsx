import { vi } from 'vitest'
import { render } from '@testing-library/react'
import MockDate from 'mockdate'
import DalmiaDalmiapuramInvoice from './dalmiaDalmiapuram'
import DalmiaKadappaInvoice from './dalmiaKadapa'
import { billNoContext } from '../../invoiceContext'
vi.mock('to-words', () => ({
    ToWords: vi.fn().mockImplementation(() => ({
        convert: vi.fn().mockReturnValue('Ten Lakhs Only')
    }))
}))

describe('Chettinad invoice component', () => {
    test('chettinad ariyalur renders correctly', () => {
        MockDate.set(new Date('2021-05-31'))
        const getProps = () => ({
            tripId: [
                {
                    overallTripId: 1,
                    tripId: 1,
                    tripName: 'LoadingToUnloading'
                }
            ],
            setLoading: vi.fn(),
            loading: false
        })
        const { asFragment } = render(
            <billNoContext.Provider
                value={{
                    invoiceValues: {
                        billNo: 'asd',
                        date: 1704886154
                    },
                    setInvoiceValues: () => {}
                }}
            >
                <DalmiaDalmiapuramInvoice {...getProps()} />
            </billNoContext.Provider>
        )
        expect(asFragment()).toMatchSnapshot()
        MockDate.reset()
    })
    test('chettinad karikali renders correctly', () => {
        MockDate.set(new Date('2021-05-31'))
        const getProps = () => ({
            tripId: [
                {
                    overallTripId: 1,
                    tripId: 1,
                    tripName: 'LoadingToUnloading'
                }
            ],
            setLoading: vi.fn(),
            loading: false
        })
        const { asFragment } = render(
            <billNoContext.Provider
                value={{
                    invoiceValues: {
                        billNo: 'asd',
                        date: 1704886154
                    },
                    setInvoiceValues: () => {}
                }}
            >
                <DalmiaKadappaInvoice {...getProps()} />
            </billNoContext.Provider>
        )
        expect(asFragment()).toMatchSnapshot()
        MockDate.reset()
    })
})
