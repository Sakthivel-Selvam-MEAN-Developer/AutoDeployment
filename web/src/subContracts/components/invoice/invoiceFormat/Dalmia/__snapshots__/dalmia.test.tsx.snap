// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Chettinad invoice component > chettinad ariyalur renders correctly 1`] = `
<DocumentFragment>
  <section
    id="dalmia_dalmiapuram_section"
  >
    <main
      class="dalmia_dalmiapuram_main"
    >
      <table
        class="border-top"
        style="width: 100%;"
      >
        <tbody>
          <tr>
            <th
              class="bg"
              colspan="5"
              style="padding: 10px;"
            >
              TAX INVOICE
            </th>
          </tr>
          <tr>
            <td>
              PAN No
            </td>
            <td>
              ABBFM2821M
            </td>
            <td
              colspan="2"
            >
              Phone/Cell No
            </td>
            <td>
              8220018402
            </td>
          </tr>
          <tr>
            <td>
              GST No
            </td>
            <td>
              33ABBFM2821M2ZD
            </td>
            <td
              colspan="2"
            >
              E Mail ID
            </td>
            <td>
              magnumlogistics.erd@gmail.com
            </td>
          </tr>
          <tr>
            <td>
              PF Code
            </td>
            <td>
              Nil
            </td>
            <td
              colspan="2"
            >
              Bill No
            </td>
            <td>
              asd
            </td>
          </tr>
          <tr>
            <td>
              ESI Code
            </td>
            <td>
              Nil
            </td>
            <td
              colspan="2"
            >
              Bill Date
            </td>
            <td>
              10/01/2024
            </td>
          </tr>
          <tr>
            <td>
              Vendor Code
            </td>
            <td>
              1190781803
            </td>
            <td
              colspan="2"
            >
              Running / Final
            </td>
            <td>
              2024-2025
            </td>
          </tr>
          <tr>
            <td
              colspan="2"
              rowspan="5"
            >
              To 
              <br />
              The Authorised Officer 
              <br />
              DALMIA CEMENT (BHARAT) LTD
              <br />
              Dalmiapuram - 621651
              <br />
              GST No 33AADCA9414C1Z6
            </td>
            <td
              colspan="2"
            >
              PO/WO No Date
            </td>
            <td />
          </tr>
          <tr>
            <td
              colspan="2"
            >
              Cost Centre / Asset Code
            </td>
          </tr>
          <tr>
            <td
              rowspan="2"
            >
              Billing Period
            </td>
            <td>
              From
            </td>
            <td>
              01/01/1970
            </td>
          </tr>
          <tr>
            <td>
              To
            </td>
            <td>
              01/01/1970
            </td>
          </tr>
          <tr>
            <td
              colspan="2"
            >
              Entry Sheet No. & Date
            </td>
            <td />
          </tr>
          <tr>
            <td>
              Nature Of Work:
            </td>
            <td
              colspan="4"
            >
              Cement Transportation Logistics work carried out in DPM to variours dist
            </td>
          </tr>
          <tr>
            <td>
              HSN/SAC CODE: 996791
            </td>
            <td>
              Service description :
            </td>
            <td
              colspan="3"
            >
              Goods Transports Agency Serivice For Road Transport
            </td>
          </tr>
          <tr
            class="border-top"
          >
            <td
              class="border-top"
              colspan="5"
            >
              Dear Sir,
              <br />
              <p>
                         Kindly process this invoice raised by our establishment, namely MAGNUM LOGISTICS for the service rendered to your organisation as required under the terms of WO/ Agreement and settle the bill amount.
              </p>
            </td>
          </tr>
        </tbody>
      </table>
      <table
        class="table2"
        style="width: 100%;"
      >
        <tbody>
          <tr
            class="bg"
            style="text-align: center;"
          >
            <th
              style="padding: 10px;"
            >
              GST No
            </th>
            <th
              style="padding: 10px;"
            >
              Description
            </th>
            <th
              style="padding: 10px;"
            >
              UM
            </th>
            <th
              style="padding: 10px;"
            >
              QTY
            </th>
            <th
              style="padding: 10px;"
            >
              Rate
            </th>
            <th
              style="padding: 10px;"
            >
              Amount(Rs)
            </th>
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
            />
            <td
              class="border tc"
            >
              As Per Annexure
            </td>
            <td
              class="border"
            />
            <td
              class="border tc"
            >
              0
            </td>
            <td
              class="border tc"
            >
              As Per Annexure
            </td>
            <td
              class="border tr"
            >
              0
            </td>
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border tr"
            >
              -
            </td>
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border tr"
            >
              -
            </td>
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border tr"
            >
              -
            </td>
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
            />
            <td>
              Taxable Value
            </td>
            <td />
            <td
              class="tc"
            >
              0
            </td>
            <td />
            <td
              class=" tr"
            >
              0
            </td>
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
            />
            <td
              class="border"
            >
              Total Value Of Service
            </td>
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
            />
            <td
              class="border"
            >
              Add : GST Amount
            </td>
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
          </tr>
          <tr
            class="border"
            style="padding-top: 10px;"
          >
            <td
              class="border"
            />
            <td
              class="border tc"
            >
              CGST@6%
            </td>
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border tr"
            >
               0.00
            </td>
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
            />
            <td
              class="border tc"
            >
              SGST@6%
            </td>
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border tr"
            >
               0.00
            </td>
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
            />
            <td
              class="border tc"
            >
              IGST 12%
            </td>
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border tr"
            >
               0.00
            </td>
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
            />
            <td
              class="border tc"
            >
              Round Off
            </td>
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border tr"
            >
              0.00
            </td>
          </tr>
          <tr
            class="bg"
            style="text-align: right;"
          >
            <td
              colspan="4"
              style="padding: 10px;"
            >
              TOTAL Value Of Invoice
            </td>
            <td
              style="padding: 10px;"
            />
            <td
              style="padding: 10px 5px;"
            >
              0.00
            </td>
          </tr>
        </tbody>
      </table>
      <table
        class="border-bottom"
        style="width: 100%;"
      >
        <tbody>
          <tr
            class="border-bottom"
          >
            <td
              class="border-bottom"
              style="padding: 10px;"
            >
              Total invoice Amount in Words :
            </td>
            <td
              class="border-bottom"
              colspan="2"
              style="padding: 10px;"
            >
              Ten Lakhs Only
            </td>
          </tr>
          <tr>
            <td
              colspan="2"
              style="padding: 10px;"
            >
              GST Payable Under Reverse Charges (No) :
            </td>
            <td
              style="padding: 10px;"
            >
              Certified that the particulars given above are true and Correct
            </td>
          </tr>
          <tr>
            <td
              colspan="3"
              style="padding: 10px;"
            >
              We have taken registration under the CGST Act,2017 and have exercised the option to pay tax on services of GTA in relation to transport of Goods supplied by us during the financial year 2024-2025 under the forward charge: Yes
            </td>
          </tr>
        </tbody>
      </table>
      <table
        class="border-bottom"
        style="width: 100%;"
      >
        <tbody>
          <tr
            class="border-bottom"
          >
            <th
              class="border-bottom tc"
              style="padding: 10px;"
            >
              Bank Details:
            </th>
            <th
              class="border-bottom"
            >
              For Magnum Logistics
            </th>
          </tr>
          <tr
            class="border-top"
          >
            <td
              class="border-top"
            >
              Bank Name: Indian Overseas Bank 
              <br />
              ACCount Name: MAGNUM LOGISTICS 
              <br />
              A/C No: 159433000055555
              <br />
              Branch : Kollampalayam
              <br />
            </td>
            <td
              class="tc"
            >
              <div
                style="padding: 50px;"
              />
            </td>
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
            >
              IFSC code : IOBA0001594
            </td>
            <td
              style="text-align: center;"
            >
              Authorised Signatory
            </td>
          </tr>
        </tbody>
      </table>
    </main>
  </section>
  <hr
    style="margin: 0px 20px;"
  />
  <section
    id="dalmia_annexure_section"
  >
    <div
      class="dalmia_annexure_main"
    >
      <div
        class="dalmia-head df"
      >
        <div>
          <table
            style="text-align: left; margin-bottom: 50px; width: 300px;"
          >
            <tbody>
              <tr>
                <th>
                  FROM
                </th>
              </tr>
              <tr>
                <th>
                  MAGNUM LOGISTICS
                </th>
              </tr>
              <tr
                style="height: 100px;"
              >
                <td>
                  NA
                </td>
              </tr>
              <tr>
                <td>
                  PAN NO:-
                </td>
              </tr>
              <tr>
                <td>
                  GSTIN:- 33ABBFM2821M2ZD
                </td>
              </tr>
            </tbody>
          </table>
        </div>
        <div>
          <table
            style="text-align: left; width: 350px;"
          >
            <tbody>
              <tr>
                <th>
                  TO
                </th>
              </tr>
              <tr>
                <th>
                  DCBL FACTORY-DALMIAPURAM
                </th>
              </tr>
              <tr
                style="height: 70px;"
              >
                <td>
                  TAMILNADU
                </td>
              </tr>
            </tbody>
          </table>
          <p>
            <b>
              Bill Sequence: I000-0000126586
            </b>
          </p>
        </div>
      </div>
      <div
        style="clear: left; text-align: left;"
      >
        <table
          style="width: 320px;"
        >
          <tbody>
            <tr>
              <th>
                Bill No: asd
              </th>
            </tr>
            <tr>
              <th>
                Bill Date: 10/01/2024
              </th>
            </tr>
            <tr>
              <th>
                For Dist. Channel: STOCK TRANSFER
              </th>
            </tr>
            <tr
              style="height: 50px;"
            >
              <th>
                WO Number: 
              </th>
            </tr>
          </tbody>
        </table>
      </div>
      <br />
      <br />
      <div>
        <b>
          BEING TRANSPORTATION CHARGES OF CEMENT FROM DCBL FACTORY-DALMIAPURAM TO DIFFERENT DESTINATIONS
        </b>
        <table
          style="width: 100%; margin-top: 20px;"
        >
          <tbody>
            <tr>
              <th>
                S.NO
              </th>
              <th>
                BILL DATE
              </th>
              <th>
                DO NO
              </th>
              <th>
                SHIPPMENT NO
              </th>
              <th>
                DESTINATION
              </th>
              <th>
                TRUCK NO
              </th>
              <th>
                QUANTITY
              </th>
              <th>
                RATE
              </th>
              <th>
                AMOUNT
              </th>
              <th>
                SHORTAGE (KG)
              </th>
              <th>
                REMARKS
              </th>
            </tr>
            <tr
              style="font-weight: bold;"
            >
              <td />
              <td />
              <td />
              <td />
              <td />
              <td
                class="tc"
              >
                TOTAL VALUE
              </td>
              <td
                class="tc"
              >
                0
              </td>
              <td />
              <td
                class="tc"
              >
                0
              </td>
              <td
                class="tc"
              >
                0
              </td>
              <td />
            </tr>
          </tbody>
        </table>
      </div>
      <div
        class="footer df "
        style="align-items: end;"
      >
        <div>
          <p>
            No. Of Challans : 0
          </p>
          <p>
            Rupees in Word :
            <b>
               Ten Lakhs Only
            </b>
          </p>
          <p>
            <b>
              GST will be paid by MAGNUM LOGISTICS
            </b>
          </p>
          <p>
            <b>
              GSTIN : 33ABBFM2821M2ZD
            </b>
          </p>
          <p>
            Whether Tax payment on Reverse Charge Basis : 
            <span>
              No
            </span>
          </p>
        </div>
        <p
          style="margin-left: 20px;"
        >
          <b>
            FOR MAGNUM LOGISTICS
          </b>
        </p>
      </div>
    </div>
  </section>
</DocumentFragment>
`;

exports[`Chettinad invoice component > chettinad karikali renders correctly 1`] = `
<DocumentFragment>
  <section
    id="dalmia_kadappa_section"
  >
    <main
      class="dalmia_kadappa_main"
    >
      <table
        class="border-top"
        id="table1"
        style="width: 100%;"
      >
        <tbody>
          <tr
            style="background-color: rgb(242, 242, 242);"
          >
            <th
              colspan="6"
            >
              INVOICE
            </th>
          </tr>
          <tr>
            <td>
              GSTIN :
            </td>
            <td>
              33ABBFM2821M2ZD
            </td>
            <td>
              PAN
            </td>
            <td>
              ABBFM2821M
            </td>
            <td>
              Inv No.
            </td>
            <td>
              asd
            </td>
          </tr>
          <tr>
            <td>
              State :
            </td>
            <td>
              Tamil Nadu
            </td>
            <td>
              State Code
            </td>
            <td>
              33
            </td>
            <td>
              Date
            </td>
            <td>
              10/01/2024
            </td>
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
              colspan="6"
            >
              Details of Receiver/ Billed to:
            </td>
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
              colspan="6"
            >
              Name : M/s. Dalmia Cement(Bharat) Limited
            </td>
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
              colspan="6"
            >
              Address : Chinnakomerla Village Jammalmadugu, Kadappa District, Andra Pradesh - 516434
            </td>
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
              colspan="6"
            >
              GSTIN : 37AADCA9414C1ZY
            </td>
          </tr>
        </tbody>
      </table>
      <table
        class="table2"
        style="width: 100%;"
      >
        <tbody>
          <tr
            class="border tc"
          >
            <td
              class="border tl"
            >
              State : Andhra Pradesh
            </td>
            <td>
              State Code
            </td>
            <td>
              37
            </td>
            <td>
              PAN
            </td>
            <td
              colspan="2"
            >
              AADCA9414C
            </td>
          </tr>
          <tr
            class="tc"
          >
            <td
              class="p-15"
            >
              Name of Product/Service
            </td>
            <td
              class="p-15"
            >
              To (Place)
            </td>
            <td
              class="p-15"
            >
              SAC
            </td>
            <td
              class="p-15"
            >
              Qty in MT
            </td>
            <td
              class="p-15"
            >
              Rate/MT
            </td>
            <td
              class="p-15"
            >
              Amount
            </td>
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
            >
              Being Transport Charges
            </td>
            <td
              class="border"
            />
            <td
              class="border tc"
            >
              996791
            </td>
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
            >
              Material: Cement
            </td>
            <td
              class="border tc"
            >
              As Per Annexure
            </td>
            <td
              class="border"
            />
            <td
              class="border tc"
            >
              0
            </td>
            <td
              class="border tc"
            >
              As Per Annexure
            </td>
            <td
              class="border tr"
            >
              0.00
            </td>
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
            >
              Period From : 01/01/1970
            </td>
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
            >
              Period To : 01/01/1970
            </td>
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
          </tr>
          <tr
            class="border"
          >
            <td
              class="border tc"
            >
              (Detailed As per Annexure Enclosed)
            </td>
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
            <td
              class="border"
            />
          </tr>
          <tr>
            <td
              class="tc"
              colspan="2"
            >
              Total
            </td>
            <td />
            <td
              class="tc"
            >
              0
            </td>
            <td />
            <td
              class="tr"
            >
              0.00
            </td>
          </tr>
          <tr>
            <td
              colspan="2"
            >
              Total Invoice Amount in Words:
            </td>
            <td
              colspan="3"
            >
              Total Amount Before Tax
            </td>
            <td
              class="tr"
            >
              0.00
            </td>
          </tr>
          <tr>
            <td
              class="tc"
              colspan="2"
              rowspan="3"
            >
              Ten Lakhs Only
            </td>
            <td
              colspan="2"
            >
              Add : CGST
            </td>
            <td />
            <td
              class="tr"
            >
              -
            </td>
          </tr>
          <tr>
            <td
              colspan="2"
            >
              Add : SGST
            </td>
            <td />
            <td
              class="tr"
            >
              -
            </td>
          </tr>
          <tr>
            <td
              colspan="2"
            >
              Add : IGST
            </td>
            <td>
              12.0%
            </td>
            <td
              class="tr"
            >
              0.00
            </td>
          </tr>
          <tr>
            <td
              class="tc"
              colspan="2"
              rowspan="3"
            >
              Certified that the particulars given above are true and correct.
            </td>
            <td
              colspan="3"
            >
              Tax Amount : IGST @12%
            </td>
            <td
              class="tr"
            >
              0.00
            </td>
          </tr>
          <tr>
            <td
              colspan="3"
            >
              Round off
            </td>
            <td
              class="tr"
            >
              0.00
            </td>
          </tr>
          <tr>
            <td
              colspan="3"
            >
              Total Amount After Tax
            </td>
            <td
              class="tr"
            >
              0.00
            </td>
          </tr>
          <tr>
            <td
              colspan="6"
            >
              We have taken registration under the CGST Act,2017 and have exercised the option to pay tax on services of GTA in relation to transport of Goods supplied by us during the financial year 2024-2025 under the forward charge.
            </td>
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
              colspan="2"
              style="text-decoration: underline;"
            >
              Bank Details:
            </td>
            <td
              class="tc"
              colspan="4"
              rowspan="5"
            />
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
              colspan="2"
            >
              Bank Name: Indian Overseas Bank
            </td>
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
              colspan="2"
            >
              Branch : Kollampalayam
            </td>
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
              colspan="2"
            >
              A/C No: 159433000055555
            </td>
          </tr>
          <tr
            class="border"
          >
            <td
              class="border"
              colspan="2"
            >
              IFSC code : 10BA0001594
            </td>
          </tr>
        </tbody>
      </table>
    </main>
  </section>
  <hr
    style="margin: 0px 20px;"
  />
  <section
    id="dalmia_annexure_section"
  >
    <div
      class="dalmia_annexure_main"
    >
      <div
        class="dalmia-head df"
      >
        <div>
          <table
            style="text-align: left; margin-bottom: 50px; width: 300px;"
          >
            <tbody>
              <tr>
                <th>
                  FROM
                </th>
              </tr>
              <tr>
                <th>
                  MAGNUM LOGISTICS
                </th>
              </tr>
              <tr
                style="height: 100px;"
              >
                <td>
                  NA
                </td>
              </tr>
              <tr>
                <td>
                  PAN NO:-
                </td>
              </tr>
              <tr>
                <td>
                  GSTIN:- 33ABBFM2821M2ZD
                </td>
              </tr>
            </tbody>
          </table>
        </div>
        <div>
          <table
            style="text-align: left; width: 350px;"
          >
            <tbody>
              <tr>
                <th>
                  TO
                </th>
              </tr>
              <tr>
                <th>
                  DCBL FACTORY-DALMIAPURAM
                </th>
              </tr>
              <tr
                style="height: 70px;"
              >
                <td>
                  TAMILNADU
                </td>
              </tr>
            </tbody>
          </table>
          <p>
            <b>
              Bill Sequence: I000-0000126586
            </b>
          </p>
        </div>
      </div>
      <div
        style="clear: left; text-align: left;"
      >
        <table
          style="width: 320px;"
        >
          <tbody>
            <tr>
              <th>
                Bill No: asd
              </th>
            </tr>
            <tr>
              <th>
                Bill Date: 10/01/2024
              </th>
            </tr>
            <tr>
              <th>
                For Dist. Channel: STOCK TRANSFER
              </th>
            </tr>
            <tr
              style="height: 50px;"
            >
              <th>
                WO Number: 
              </th>
            </tr>
          </tbody>
        </table>
      </div>
      <br />
      <br />
      <div>
        <b>
          BEING TRANSPORTATION CHARGES OF CEMENT FROM DCBL FACTORY-DALMIAPURAM TO DIFFERENT DESTINATIONS
        </b>
        <table
          style="width: 100%; margin-top: 20px;"
        >
          <tbody>
            <tr>
              <th>
                S.NO
              </th>
              <th>
                BILL DATE
              </th>
              <th>
                DO NO
              </th>
              <th>
                SHIPPMENT NO
              </th>
              <th>
                DESTINATION
              </th>
              <th>
                TRUCK NO
              </th>
              <th>
                QUANTITY
              </th>
              <th>
                RATE
              </th>
              <th>
                AMOUNT
              </th>
              <th>
                SHORTAGE (KG)
              </th>
              <th>
                REMARKS
              </th>
            </tr>
            <tr
              style="font-weight: bold;"
            >
              <td />
              <td />
              <td />
              <td />
              <td />
              <td
                class="tc"
              >
                TOTAL VALUE
              </td>
              <td
                class="tc"
              >
                0
              </td>
              <td />
              <td
                class="tc"
              >
                0
              </td>
              <td
                class="tc"
              >
                0
              </td>
              <td />
            </tr>
          </tbody>
        </table>
      </div>
      <div
        class="footer df "
        style="align-items: end;"
      >
        <div>
          <p>
            No. Of Challans : 0
          </p>
          <p>
            Rupees in Word :
            <b>
               Ten Lakhs Only
            </b>
          </p>
          <p>
            <b>
              GST will be paid by MAGNUM LOGISTICS
            </b>
          </p>
          <p>
            <b>
              GSTIN : 33ABBFM2821M2ZD
            </b>
          </p>
          <p>
            Whether Tax payment on Reverse Charge Basis : 
            <span>
              No
            </span>
          </p>
        </div>
        <p
          style="margin-left: 20px;"
        >
          <b>
            FOR MAGNUM LOGISTICS
          </b>
        </p>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
